Python/Product/PythonTools/PythonTools/Repl/PythonDebugReplEvaluator.cs

```diff
@@ -134,7 +134,14 @@ internal class PythonDebugReplEvaluator :
 
             if (_activeEvaluator != null) {
                 return _activeEvaluator.ExecuteCodeAsync(text);
+            } else {
+                // TODO: Move this to DebugReplEvaluator for Experimental Debugger
+                if (CustomDebugAdapterProtocolExtension.CanUseExperimental()) {
```

Comment:
This is just a sample showing how we can send requests and get responses from the experimental debugger. We can moves this to the clone of Debug process repl evaluator which understands how to talk to the experimental debugger.

Reply:
Since you're not calling `CustomDebugAdapterProtocolExtension.Initialize`, this will always return false. Is it so the whole thing stays disabled for now?

Reply:
Or maybe the adapter host is the one calling Initialize?